scss/components/toggle.scss

```diff
@@ -95,7 +93,7 @@ $block: #{$fd-namespace}-toggle;
         border-style: solid;
         border-width: 1px;
         @include fd-var-color("border-color", $fd-toggle-background-color, --fd-color-neutral-4);
-        @include fd-var-color("background-color", $fd-toggle-background-color, --fd-color-neutral-3);
+        @include fd-var-color("background-color", fd-color("neutral", 3), --fd-color-neutral-3);
```

Comment:
This change was necessary because `$fd-toggle-background-color` = neutral 4, not 3.